apps/bbsync/query.py

```diff
@@ -6,9 +6,8 @@
 
 from collectors.bzimport.constants import ANALYSIS_TASK_PRODUCT
 from osidb.helpers import cve_id_comparator, filter_cves
-from osidb.models import Impact, PsModule
+from osidb.models import Flaw, Impact, PsModule
 from osidb.models.flaw.comment import FlawComment
-from osidb.models.flaw.flaw import Flaw
```

Comment:
That looks as an oversight, I will check it

Reply:
Mostly an oversight. Exceptions are AFFECTEDNESS_VALID_RESOLUTIONS which I decided to consider module-internal as apart from the factory (which needs the internals) it is not used outside of the module so I do not propagate it on the interface. The some in-module imports need to be done this longer way due to the cycles. Maybe with your suggested string referencing it might be OK but that is for the future.